main.tf

```diff
@@ -106,7 +106,7 @@ resource "aws_instance" "default" {
 
   key_name = "${var.ssh_key_pair}"
 
-  subnet_id = "${var.subnets[0]}"
+  subnet_id = "${var.subnet}"
 
   tags {
```

Comment:
this should be `tags = "${module.label.tags}"`

Reply:
will be fixed in https://github.com/cloudposse/terraform-aws-ec2-instance/pull/15